src/styles/card.css

```diff
@@ -0,0 +1,92 @@
+.card {
+  width: 200px;
+  box-sizing: border-box;
+
+  border: 1px solid #f3f3f3;
+  padding: 32px 16px 16px;
+}
+
+.card__img {
+  display: block;
+  width: 160px;
+  height: 134px;
+
+  margin: 0 auto;
+}
+
+.card__title {
+  display: block;
+  padding: 40px 0 4px;
+
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: #060b35;
+}
+
+.card__subtitle {
+  padding-bottom: 16px;
+
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+
+}
+
+.card__reviews {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #060b35;
+}
+
+.card__stars,
+.card__price {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.card__price {
+  padding: 24px 0 16px;
+}
+
+.card__price-title {
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 18px;
+  color: #616070;
+}
+
+.card__price-value {
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+  color: #060b35;
+}
+
+.card__button {
+  box-sizing: border-box;
+  display: block;
+  width: 100%;
+  height: 40px;
+
+  color: #fff;
+  font-weight: 700;
+  font-size: 14px;
+  text-align: center;
+  text-decoration: none;
+  line-height: 40px;
+
+  background: #00acdc;
+  border: 1px solid transparent;
+  border-radius: 5px;
+
+}
+
+.card__button:hover {
+  color: #00acdc;
+  background: #fff;
+  border-color: #f3f3f3;
```

Comment:
you don't need to add border color here, add instead of transparent in parent element, and remove from here. and border color should be #00acdc

Reply:
@GabrielMaksimovich I don't really understand how I can add a border color if I remove it from this block. It does not make sense.

Reply:
@batstolya got you,
lets look at example
your btn have border color and box-sizing, so when its not hover its like background color. but when you hover it you change only color of text and background color and you will see the color of the border 

`border: 1px solid red;
color: balck;
background: red;

 :hover {
 color: red;
 background-color: white;
 }

`

Reply:
@GabrielMaksimovich Now I really understand. This is a good solution, thanks) I'll fix it